resources/js/Pages/post/_Show.vue

```diff
@@ -103,6 +102,23 @@ export default {
             showCommentInput: this.post.comments.length > 0,
         }
     },
+    computed: {
+        teamFilterLink() {
+            return this.route("app.post.index", {team: this.post.team_id});
+        },
+        bannerFilterLink() {
+            return this.route("app.post.index", {banner: this.post.banner_id});
+        },
+        challengeFilterLink() {
+            return this.route("app.post.index", {project: this.post.challenge_id});
+        },
+        contentWithHashtags() {
+            return this.post.content.replaceAll(/#([a-zA-Z0-9_]+)/g, (match) => {
```

Comment:
We could eventually match `äöüÄÖÜß` and `-` as well:
```suggestion
            return this.post.content.replaceAll(/#([a-zA-Z0-9_äöüÄÖÜß-]+)/g, (match) => {
```

Reply:
Using `v-html` with `contentWithHashtags` opens a security issue. since anyone can make a post containing `<div onclick="alert('ok')">click me!</div>`.

We have to escape the HTML before inserting it as HTML:

```suggestion
            return escapeHTML(this.post.content).replaceAll(/#([a-zA-Z0-9_]+)/g, (match) => {
```

with an `escapeHTML` function like:

```js
function escapeHTML(str) {
	var el = document.createElement('div');
	el.textContent = str;
	return el.innerHTML;
};
```